spec/mongoid/persistable/savable_spec.rb

```diff
@@ -296,7 +296,7 @@
           # no conflicts.
           #expect(truck.atomic_updates[:conflicts]).to eq nil
 
-          expect { truck.save! }.not_to raise_error
+          expect(truck.save).to eq true
```

Comment:
@johnnyshields I am having difficulty understanding the changes in this file, could you explain them please?

Reply:
```
expect { truck.save! }.not_to raise_error
```

is equivalent to:

```
expect(truck.save).to eq true
```

Both statements verify the document saved successfully. I prefer the latter in this case; we are specifically testing that the document saved.